app/tag/tag-list-directive.html

```diff
@@ -7,10 +7,16 @@
 </button>
 
 <div class="table-responsive">
-  <table class="table table-hover">
+  <table st-table="displayedTags" st-safe-src="tags" class="table table-hover">
```

Comment:
I suggest to not introduce this `st-table` crap again as it caused ugly problems when you try to sort on something that is being queried after the DOM was loaded (and probably rendered already).